hhs_hosp/delphi_hhs/run.py

```diff
@@ -176,6 +183,16 @@ def make_signal(all_columns, sig):
             all_columns.previous_day_admission_pediatric_covid_confirmed + \
             all_columns.previous_day_admission_pediatric_covid_suspected,
         })
+    elif sig ==  POP_PROP:
+        df = pd.DataFrame({
+            "state": all_columns.state.apply(str.lower),
+            "timestamp":int_date_to_previous_day_datetime(all_columns.date),
+            "val": \
+            all_columns.previous_day_admission_adult_covid_confirmed + \
+            all_columns.previous_day_admission_adult_covid_suspected + \
+            all_columns.previous_day_admission_pediatric_covid_confirmed + \
+            all_columns.previous_day_admission_pediatric_covid_suspected,
+        })
```

Comment:
once the signal names are more normalized you'll be able to drop this block, and instead modify the conditions on the previous if's to something like

```
    if sig.startswith(CONFIRMED):
        ...
    elif sig.startswith(SUM_CONF_SUSP):
        ...
```